test/regress/cli/regress0/strings/from-int-eval.smt2

```diff
@@ -0,0 +1,3 @@
+(set-logic ALL)
```

Comment:
Can you add a status or `EXPECT`ed result?

Reply:
Added now